clients/database/postgres.go

```diff
@@ -16,12 +16,15 @@ import (
 // PostgresDatabaseConfig contains values for creating a
 // new connection to a postgres database
 type PostgresDatabaseConfig struct {
+	// DatabaseDisabled is used to disable the database, and it won't be used at all. All operations will be skipped.
+	DatabaseDisabled bool
+
 	DatabaseName                     string
 	DatabaseEndpointURL              string
 	DatabaseUsername                 string
 	DatabasePassword                 string
 	ReadTimeoutSeconds               int64
-	WriteTimeousSeconds              int64
+	WriteTimeoutsSeconds             int64
```

Comment:
should it be `WriteTimeoutSeconds` for consistency?

Reply:
agreed with @evgeniy-scherbina 👍

```suggestion
	WriteTimeoutSeconds             int64
```

thanks for refactoring this typo 😄